en/software-part-I/install-software-cli.md

```diff
@@ -82,10 +87,10 @@ Description: CP2102 USB to UART Bridge Controller - CP2102 USB to UART Bridge Co
 > (env) $ platformio run --list-targets
 > ```
 
-Now you can compile and upload the filesystem: <!-- TODO: einmal ota target, einmal usb? -->
```

Comment:
Yes sounds good!
Do you want to add this in this PR or should I merge it?

Reply:
Umm, not sure what you mean? The comment was for myself while I skimmed the text and noted a mistake, which finally didn't make it into the last PR, but I forgot to remove this comment. Can be merged IMO.

Reply:
ah lol, miss read this. Merge income :)